v_3_6_0/master_template.go

```diff
@@ -1503,6 +1476,37 @@ write_files:
         user: proxy
       name: service-account-context
     current-context: service-account-context
+- path: /etc/kubernetes/config/kubelet-config.yaml.tmpl
+  owner: root
+  permissions: 0644
+  content: |
+    kind: KubeletConfiguration
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    address: ${DEFAULT_IPV4}
+    port: 10250
+    healthzBindAddress: ${DEFAULT_IPV4}
+    healthzPort: 10248
+    clusterDNS:
+      - {{.Cluster.Kubernetes.DNS.IP}}
+    clusterDomain: {{.Cluster.Kubernetes.Domain}}
+    featureGates:
+      ExpandPersistentVolumes: true
```

Comment:
`CustomResourceSubresources: true` is required and missing. What about `PodPriority=true`?